Update guide.md (#128)
Start9Labs/bitcoind-wrapper: docs/integrations/sparrow/guide.md

## Configuring Sparrow to use the Bitcoin Core RPC

> 1. First, install the Bitcoin Core service to your StartOS from the Start9 Marketplace.

Reviewer: ```suggestion
1. Install the Bitcoin Core service to StartOS from the Start9 Marketplace, if you have not already.
```

2. Configure Bitcoin Core and allow it to begin the Initial Blockchain Download if you have not already.  You may continue even if this is still in progress, but you will need to let the sync complete before creating your first transaction.  Bitcoin Core MUST have the "Enable Wallet" feature turned on.  This is found in *Bitcoin Core > Config > Wallet*.

3. Next, [download](https://sparrowwallet.com/download/), install, and launch Sparrow.  If this is the first time you have run Sparrow, you will be guided to a screen where you will be asked to configure your Bitcoin server.  Otherwise, you can find the server setup in Preferences.  Select the option for "Bitcoin Core" as the **Server** Type:

    ![Sparrow Server Setup](./assets/sparrow-server-setup2.png "Setup Your Bitcoin Server")

> 4. From your StartOS's Bitcoin Core service page, click "Interfaces," and copy the `LAN Address` under **RPC Interface**.  Paste this into Sparrow's "URL" field, removing the `http://` prefix, as Sparrow will not accept it.  In the "Port" field, type `443`:

Reviewer: ```suggestion
4. From the Bitcoin Core service page in StartOS, click "Interfaces," and copy the `LAN Address` under **RPC Interface**.  Paste this into Sparrow's "URL" field, removing the `http://` prefix, as Sparrow will not accept it.  In the "Port" field, type `443`:
```

    ![Sparrow Server Setup](./assets/sparrow-server-setup3.png "Enter URL & Port")

    **Note**: Use port `8332` if you are using your `Tor Address`.

> 5. Return to your StartOS's Bitcoin Core service page again, and click "Properties" -> "RPC Username".  Copy the Username using the button to the right:

Reviewer: ```suggestion
5. Return to your server's Bitcoin Core service page again, and click "Properties" -> "RPC Username".  Copy the Username using the button to the right:
```
